src/internal/packager/deploy.go

```diff
@@ -141,6 +141,11 @@ func deployComponents(tempPath tempPaths, component types.ZarfComponent) {
 				utils.ValidateSha256Sum(file.Shasum, sourceFile)
 			}
 
+			// Replace temp target directories
+			if strings.Contains(file.Target, "###ZARF_TEMP###") {
+				file.Target = strings.Replace(file.Target, "###ZARF_TEMP###", tempPath.base, -1)
```

Comment:
does -1 make sense here?  Don't really see how you would have more than one zarf temp path in a single target?